build.gradle

```diff
@@ -1,18 +1,18 @@
 apply plugin: 'java'
 
-sourceCompatibility = 11
-targetCompatibility = 11
+sourceCompatibility = 17
+targetCompatibility = 17
```

Comment:
Not sure it these variables are something that Gradle needs. If they are not needed, I'd remove them.

Reply:
Based on Gradle docs:

>`sourceCompatibility`: Java version compatibility to use when compiling Java source. Default value: version of the current JVM in use.
?
>`targetCompatibility`: Java version to generate classes for. Default value: `sourceCompatibility`.

So, my vote would go for:

- Maintain `sourceCompatibility` as it explicitly states the Java version to use while compiling instead of delegating it implicitly. Why: We would like to throw an error otherwise making it worth it to mention that the actual environment does not match the expected one.
- Maintain `targetCompatibility` because it acquiring the `sourceCompatibility` value by default  does not seems like an easy assumption (at least in my case). However, I do not have any other reason apart from personal opinions here, so feel free to delete it if you prefer 😊